packages/tracing/src/browser/metrics.ts

```diff
@@ -122,10 +132,44 @@ export class MetricsInstrumentation {
 
     // Measurements are only available for pageload transactions
     if (transaction.op === 'pageload') {
+      this._trackNavigator();
+      transaction.setContexts({ browser: this._browserContext });
       transaction.setMeasurements(this._measurements);
     }
   }
 
+  /**
+   * Capture the information of the user agent.
+   */
+  private _trackNavigator(): void {
+    const navigator = window.navigator as null | (Navigator & NavigatorNetworkInformation & NavigatorDeviceMemory);
```

Comment:
We already have `getGlobalObject` here which can be reused.

Reply:
Done.